app/scripts/metamask-controller.js

```diff
@@ -623,6 +623,10 @@ module.exports = class MetamaskController extends EventEmitter {
       case '3':
         url = 'https://faucet.metamask.io/'
         break
+
+      case '42':
+        url = 'https://github.com/kovan-testnet/faucet'
```

Comment:
Could we link to an actual faucet, instead of a repo for creating your own faucet?